server/lib/flow-event.js

```diff
@@ -87,6 +87,7 @@ module.exports = (req, metrics, requestReceivedTime) => {
     }
 
     logFlowEvent(event, metrics, req);
+    sendToAmplitude(event, metrics, req);
```

Comment:
I expect it's only flow-events for the PoC, but it might still make sense to make a new collector just to keep it cleanly separated.